### ISSUE TITLE
bug fixed on tf1.5

### ISSUE THREAD

**Issue by fortunechen** [huyng/tensorflow-vgg#16]
1.` tf.pack` --->` tf.stack`
2. `tf.softmax_cross_entropy_with_logits(logits, onehot_labels, name='xentropy') `---> `tf.softmax_cross_entropy_with_logits(logits=logits, labels=onehot_labels, name='xentropy')
`
3.   add conv3_3 : `net = L.conv(net, name="conv3_3", kh=3, kw=3, n_out=256)`
